cmd/dlt/operatorrole/cmd.go

```diff
@@ -270,7 +273,22 @@ func run(cmd *cobra.Command, _ []string) {
 			r.Reporter.Errorf("There was an error getting the policy: %v", err)
 			os.Exit(1)
 		}
-		commands := buildCommand(r, foundOperatorRoles, policyMap, arbitraryPolicyMap, managedPolicies)
+
+		// Get HCP shared vpc policy details if the user is deleting roles related to HCP shared vpc
+		policiesOutput := make([]*iam.GetPolicyOutput, 0)
+		if roles.CheckIfRolesAreHcpSharedVpc(r, foundOperatorRoles) &&
+			confirm.Prompt(true, "Create commands to delete Hosted CP shared VPC policies?") {
+			for _, role := range foundOperatorRoles {
+				policies, err := r.AWSClient.GetPolicyDetailsFromRole(awssdk.String(role))
+				policiesOutput = append(policiesOutput, policies...)
+				if err != nil {
+					r.Reporter.Infof("There was an error getting details of policies attached to role '%s': %v",
```

Comment:
why not use r.Reporter.Errorf() here?

Reply:
It is better to not error here as customers are not expected to have set up the right permissions for our aws client to work with fetching policies, so we want to provide the normal role deletion commands even if the client is unable to fetch the policies

Reply:
Should it be a warn instead then?